### PR TITLE
Incoming CIDR should be a string in Spaces api and facade, not a SubnetTag.

### PR DESCRIPTION
**Pull request by hmlanigan**
## Description of change

Change Spaces api and facade to accept a CIDR as a string to be verified, not a names.v2 SubnetTag.  The facade version was incremented to 5

## QA steps

1. juju bootstrap maas.
2. juju subnets
2. juju add-space testme <valid network cidr> - succeeds
2. use older client against new controller:
2. juju add-space testme <valid network cidr>  - succeeds
3. juju spaces to verify

### CODE REVIEW

File: api/spaces/spaces.go [hunk 1]
```diff
@@ -31,27 +31,18 @@ func NewAPI(caller base.APICallCloser) *API {
...
-	return params.CreateSpaceParams{
-		SpaceTag:   spaceTag,
-		SubnetTags: subnetTags,
-		Public:     public,
-	}
-}
-
 // CreateSpace creates a new Juju network space, associating the
 // specified subnets with it (optional; can be empty).
 func (api *API) CreateSpace(name string, subnetIds []string, public bool) error {
```
Comment (manadart) [inline api/spaces/spaces.go]: Shall we rename the `subnetIds` argument?

Comment (manadart) [inline api/spaces/spaces.go]: Shall we rename the `subnetIds` argument?

Comment (hmlanigan) [reply api/spaces/spaces.go]: yes, missed this one.  ty

**Comment by hmlanigan** [comment]
Updated to allow for new client old controller.

**Pull request merged**